python_modules/dagster/dagster_tests/daemon_tests/test_backfill.py

```diff
@@ -82,6 +83,7 @@
     step_did_not_run,
     step_failed,
     step_succeeded,
+    wait_for_futures,
```

Comment:
Having a test that ensures that a backfill doesn't get processed if it's already being processed in a thread would be good to ensure we don't accidentally regress

Reply:
Would [37f4a69](https://github.com/dagster-io/dagster/pull/26189/commits/37f4a696a6f32a26e90591826f6a5e260ecfd1ef) work?

Reply:
Updated as per later comments.